assemble.py

```diff
@@ -391,5 +394,10 @@ def get_args():
         os.makedirs(dst_dir, exist_ok=True)
         cmd('tar', '-cf', os.path.join(dst_dir, target + '-apps.tar'), '-C', apps_desc.dir, '.')
 
+    # Cleanup the fetched images
+    if os.path.exists(fetch_dir):
+        logger.info(f'Removing `{fetch_dir}` directory Apps were fetched to...')
+        shutil.rmtree(fetch_dir, ignore_errors=True)
```

Comment:
The `image` sub-dir has some odd attributes after been used by `dockerd` so we need to ignore the removal error. It doesn't impact removal anyhow, all data are removed.